cloud/backend/cf-d1/cf-hono-server.ts

```
@@ -0,0 +1,347 @@
+      if (idx >= 0) {
+        this.notWebSockets.splice(idx, 1);
+      }
+      return;
+    }
+    const found = this.getConns().find((i) => ps.cloud.qsidEqual(i.conn, conn));
+    if (!found) {
+      return;
+    }
+    // console.log("removeConn", this.id, conn);
+    const s = found.ws.raw?.deserializeAttachment();
+    delete s.conn;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)